### PR TITLE
Update simpler-wick.sty

### PR DESCRIPTION
**Pull request by aeberso2**
Added in a new environment for the creation of bottom contractions, called \bwick #4 
Also added the option to adjust line width of the contraction lines if desired and adjusted some spacing. #5 

Changes further detailed in my comments in the update

### CODE REVIEW

**Comment by JP-Ellis** [comment]
Hey!  Thanks for the contribution!  I know I should've gotten to these issues earlier... but then here you are :)

With regards to the adjustment of the line width, I'm pretty happy with the changes you've made.  The only thing is that I don't think the option should be called `conwidth` as that seems a bit obscure.  I think it would make more sense to call that `line width` as that is consistent with Ti*k*Z, and it is quite clear what it refers to even if someone didn't read the documentation.

Also, documentation needs to be added for this new option and show use-cases of the variable line width.  

I'll fix the Travis script to make sure it works again so that the pull request can be merged.  This will require you to rebase you request.

----

With regards to bottom contractions, I would rather not use a new environment.  Instead, I think it would be better to introduce a `\c*` command that works within the existing `\wick` environment which produces a bottom contraction.  (This is because in LaTeX, there is already a convention of starred functions denoting a close alternative of the original functions).

In fact, I think there should be also `\wick*` option which would flip all contractions.  That is, in `\wick`, `\c` and `\c*` create Wick contractions above and beneath the symbols respectively; and in `\wick*`, `\c` and `\c*` would create Wick contractions beneath and above the symbols respectively.

There should also be a global option that causes all contractions to flip from above to beneath.

This will require a few more changes, including two separate counters for the number of above and below contractions.  I'd be happy to implement that this weekend.  In the meantime, you'll have to remove those specific changes from this pull request.

**Comment by aeberso2** [comment]
Yes, lol I'm not always the best at naming conventions, and I agree your environment idea sounds much better, I didn't really know how to implement it in a more efficient, hence the separate environment. Anyways, I have updated those changes to the pull request, changing `conwidth` to `line width` and removing the other environment. Also, I'm a bit new to how github works with pull requests, rebasing and such, so please let me know if I need to do anything additional to the commit.

File: simpler-wick.sty [hunk 1]
```diff
@@ -34,9 +34,9 @@
   sep=3pt,
   offset/.store in=\swick@offset,
   offset=1em
-  % L. new option for specifying line width, currently set to the TikZ default of 0.4pt 
-  conwidth/.store in=\swick@conwidth,
-  conwidth=0.4pt}
+  % L. new option for specifying line width, currently set to the TikZ default of 0.4pt
```
Comment (JP-Ellis) [inline simpler-wick.sty]: No need for this line, the main documentation should cover that.

File: simpler-wick.sty [hunk 2]
```diff
@@ -193,7 +193,7 @@
   \tikz[remember picture, overlay]
   % L. added contraction width option, also adjusted the start of the contraction line to be a bit lower, 
   % while initially fine, it looked a bit off for the bottom contraction so I altered this to be consistent
```
Comment (JP-Ellis) [inline simpler-wick.sty]: Similarly, this comment is not needed in the final code (fine in a pull request, but it doesn't need to stay in the long run)

File: simpler-wick.sty [hunk 3]
```diff
@@ -255,50 +255,17 @@
...
-		simplerwick,
-		#1}
-	% Define the variables and commands
-	\swick@cond@reset
-	\swick@count=0
-	\def\swick@max{0}
-	\def\b{\swick@sbcon} % this doesn't have to be \b, originally I had defined it so it could be used in the above environment, however
-  % the vertical spacing is not right due to the vbox, so I had to create a new environment so the distinction is no longer necessary 
-  % this might be resolved with some conditional statements, but this works for now.
+% L. Remant section,
```
Comment (JP-Ellis) [inline simpler-wick.sty]: The remaining code here is just commented out, so it could be remove from the pull request.

**Comment by JP-Ellis** [comment]
Thanks for the changes!  I've just made a few small comments (mostly, extra comments in the code which don't really need to be there).

Also, you'll need to make sure that pull request compiles fine on Travis-CI before I can merge it; your latest commit doesn't actually compile unfortunately.  You can view the build output [here](https://travis-ci.org/JP-Ellis/simpler-wick/builds/236403851).

**Comment by aeberso2** [comment]
Found the issue, forget a comma after `offset=1em`, also took out the comments which were no longer necessary

**Pull request merged**